src/lib/components/ratio/ColorIssues.svelte

```diff
@@ -29,10 +29,21 @@
 </details>
 
 <style lang="scss">
-  dl {
-    display: block;
+  .known-issues {
+    grid-area: knownissues;
+    padding: var(--shim) var(--double-gutter);
+  }
+  .issues-list {
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-auto-rows: auto;
```

Comment:
@stacyk this grid (and the paragraph spacing) takes into account additional [issues being added later](https://github.com/oddbird/oddcontrast/pull/145#issuecomment-1854366303).